Options.php

```diff
@@ -59,8 +59,9 @@ public function get_key() {
 	 * @return mixed Whatever is in those fields.
 	 */
 	public function get( $field = null, $default = null ) {
+		$option_array = get_option( $this->key, array() );
+		$option_array = gettype($option_array) != "array" ? [] : $option_array;
```

Comment:
It would probably be simpler to just cast to array.

    $option_array = (array) get_option( $this->key, array() );

or

    $data = array_merge( $this->defaults, (array) get_option( $this->key, array() ) );